uiowa/uiowa
# Update simplei colors to match brand.

Resolves #1556 

I swapped test and prod because I thought we mentioned toning down the prod indicator since most customers do not interact with local, dev, or test instances. Thoughts?

Prod
![image](https://user-images.githubusercontent.com/1068523/81434396-b1c01c00-912b-11ea-9f37-d3296489b0cc.png)

Test/Stage
![image](https://user-images.githubusercontent.com/1068523/81434431-bbe21a80-912b-11ea-90fc-096d4960f54d.png)

Dev
![image](https://user-images.githubusercontent.com/1068523/81434450-c4d2ec00-912b-11ea-845a-9a61c4f57e3e.png)

Local
![image](https://user-images.githubusercontent.com/1068523/81434475-cd2b2700-912b-11ea-8565-dcb8d64a15cb.png)

How to test
Check out the branch locally and override the `$env` variable in sitenow.settings.php to dev/test/prod/local. Refresh. Don't commit.
